### DISCUSSION TITLE
[1604030] V Rising/夜族崛起 [本体]

### DISCUSSION THREAD

**Discussion by ren926** [pjy612/SteamManifestCache#561] [Share]
`appid:1604030`

`1604031;e51d4f407e118a5477cab33cf9cd3340057f5aa111772951f47d76b05b594606`
[v ring 清单和key.zip](https://github.com/pjy612/SteamManifestCache/files/14819576/v.ring.key.zip)

email;[1821848667@qq.com](mailto:1821848667@qq.com)
code:84A38AF8CF13E39C

**Comment by ZEERDEER** [comment]
游戏名：V Rising
